bc_obps/reporting/models/reporting_field.py

```diff
@@ -10,9 +10,12 @@ class ReportingField(BaseModel):
         max_length=1000, db_comment="Name of field needed for the related configuration element."
     )
     field_type = models.CharField(max_length=1000, db_comment="Type definition for field.")
+    field_units = models.CharField(
+        max_length=1000, blank=True, null=True, db_comment="Units of measurement relating to the field."
+    )
 
     def serialize(self) -> "dict[str,str]":
-        return {"fieldName": self.field_name, "fieldType": self.field_type}
+        return {"fieldName": self.field_name, "fieldType": self.field_type, "fieldUnits": self.field_units}
 
     class Meta:
```

Comment:
This table could have a unique constraint on field_name, field_type, field_units to help developers as they are adding new records to ensure that we don't end up duplicating things in this table